int32 type
EcatRxPdo rxPdo
EcatSdoData sdo
EcatCommand command